### PR TITLE
Release kibana 2.7.0-6.8.2 (automated commit)



### PR DESCRIPTION
**Pull request by mesosphere-teamcity**
Release kibana 2.7.0-6.8.2 (automated commit)

Description:
Source URL: https://infinity-artifacts.s3.amazonaws.com/permanent/kibana/assets/2.7.0-6.8.2/stub-universe-kibana.json

Changes between revisions 1100 => 1200:
0 files added: []
0 files removed: []
4 files changed:

```
--- 1100/config.json
+++ 1200/config.json
@@ -107,6 +107,11 @@
           "type": "integer",
           "default": 2048
         },
+        "xpack_enabled": {
+          "description": "[DEPRECATED] X-Pack is enabled by default on Kibana 6.3+. Changing this setting now has no effect.",
+          "type": "boolean",
+          "default": true
+        },
         "custom_kibana_yml": {
           "description": "Custom YAML to be appended to kibana.yml. This field must be base64 encoded.",
           "type": "string",
@@ -115,11 +120,6 @@
             "type": "application/x-yaml"
           },
           "default": ""
-        },
-        "xpack_enabled": {
-          "description": "[DEPRECATED in 2.5.0-6.3.2] X-Pack is enabled by default. Changing this setting has no effect.",
-          "type": "boolean",
-          "default": true
         }
       },
       "required": [
--- 1100/marathon.json.mustache
+++ 1200/marathon.json.mustache
@@ -38,7 +38,7 @@
     "DCOS_PACKAGE_FRAMEWORK_NAME": "{{service.name}}"
   },
   "env": {
-    "ELASTIC_VERSION": "7.6.0",
+    "ELASTIC_VERSION": "6.8.1",
     "FRAMEWORK_NAME": "{{service.name}}",
     "FRAMEWORK_USER": "{{service.user}}",
     "KIBANA_PLUGINS": "{{kibana.plugins}}",
@@ -62,13 +62,13 @@
       "cache": true
     },
     {
-      "uri": "{{resource.assets.uris.init-sh}}",
+      "uri": "https://infinity-artifacts-ci.s3.amazonaws.com/autodelete7d/kibana/20200601-114631-66kytXDpebf1yVYj/nginx.conf.tmpl",
+      "cache": true
+    },
+    {
+      "uri": "https://infinity-artifacts-ci.s3.amazonaws.com/autodelete7d/kibana/20200601-114631-66kytXDpebf1yVYj/init.sh",
       "cache": true,
       "executable": true
-    },
-    {
-      "uri": "{{resource.assets.uris.nginx-conf-tmpl}}",
-      "cache": true
     }
   ],
   "upgradeStrategy":{
--- 1100/package.json
+++ 1200/package.json
@@ -1,16 +1,16 @@
 {
   "packagingVersion": "4.0",
   "upgradesFrom": [
-    "3.1.1-7.6.0"
+    "2.7.0-6.8.1"
   ],
   "downgradesTo": [
-    "3.1.1-7.6.0"
+    "2.7.0-6.8.1"
   ],
-  "minDcosReleaseVersion": "1.11",
+  "minDcosReleaseVersion": "1.9",
   "name": "kibana",
-  "version": "3.1.2-7.6.0",
+  "version": "2.7.0-6.8.2",
   "maintainer": "support@mesosphere.io",
-  "description": "Kibana 7, and X-Pack",
+  "description": "Kibana 6, and X-Pack",
   "selected": true,
   "framework": true,
   "tags": [
@@ -19,9 +19,9 @@
     "kibana",
     "x-pack"
   ],
-  "preInstallNotes": "Default configuration requires 1 agent node with: 0.5 CPU | 2048 MB MEM",
+  "preInstallNotes": "This DC/OS service is currently a beta candidate undergoing testing as part of a formal beta test program. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.\n\nDefault configuration requires 1 agent node with: 0.5 CPU | 2048 MB MEM\n\nContact Mesosphere before deploying this beta candidate service. Product support is available to approved participants in the beta test program.",
   "postInstallNotes": "The DC/OS Kibana service is being installed!\n\n\tDocumentation: https://docs.mesosphere.com/services/elastic/\n\tIssues: https://docs.mesosphere.com/support/",
-  "postUninstallNotes": "The DC/OS Kibana service is being uninstalled.\n\nFor DC/OS versions from 1.11 no further action is required. For older DC/OS versions follow the instructions at https://docs.mesosphere.com/services/elastic/uninstall to remove any persistent state if required.",
+  "postUninstallNotes": "The DC/OS Kibana service is being uninstalled.\n\nFor DC/OS versions from 1.10 no further action is required. For older DC/OS versions follow the instructions at https://docs.mesosphere.com/services/elastic/uninstall to remove any persistent state if required.",
   "licenses": [
     {
       "name": "Apache License Version 2.0",
@@ -32,5 +32,5 @@
       "url": "https://raw.githubusercontent.com/elastic/kibana/master/x-pack/README.md"
     }
   ],
-  "lastUpdated": 1585735921
+  "lastUpdated": 1591011992
 }--- 1100/resource.json
+++ 1200/resource.json
@@ -1,9 +1,9 @@
 {
   "assets": {
     "uris": {
-      "kibana-tar-gz": "https://downloads.mesosphere.com/elastic/assets/kibana-7.6.0-linux-x86_64.tar.gz",
-      "init-sh": "https://downloads.mesosphere.com/kibana/assets/3.1.2-7.6.0/init.sh",
-      "nginx-conf-tmpl": "https://downloads.mesosphere.com/kibana/assets/3.1.2-7.6.0/nginx.conf.tmpl"
+      "kibana-tar-gz": "https://downloads.mesosphere.com/elastic/assets/kibana-6.8.1-linux-x86_64.tar.gz",
+      "init-sh": "https://downloads.mesosphere.com/kibana/assets/2.7.0-6.8.2/nginx.conf.tmpl",
+      "nginx-conf-tmpl": "https://downloads.mesosphere.com/kibana/assets/2.7.0-6.8.2/init.sh"
     },
     "container": {
       "docker": {
```

### CODE REVIEW

File: repo/packages/K/kibana/1200/marathon.json.mustache [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    {{/service.virtual_network_enabled}}
+    "CUSTOM_YAML_BLOCK_BASE64": "{{kibana.custom_kibana_yml}}"
+  },
+  "fetch": [
+    {
+      "uri": "{{resource.assets.uris.kibana-tar-gz}}",
+      "cache": true
+    },
+    {
+      "uri": "https://infinity-artifacts-ci.s3.amazonaws.com/autodelete7d/kibana/20200601-114631-66kytXDpebf1yVYj/nginx.conf.tmpl",
```
Comment (kaiwalyajoshi) [inline repo/packages/K/kibana/1200/marathon.json.mustache]: This URI has a `autodelete7d` filed and will expire, we cannot release with this, same for other one below.
Can you check if the stubs we used to create this also had these `autodelete7d` fields? We'll need to verify if our tooling didn't catch instances of this.

**Review by kaiwalyajoshi — DISMISSED** [review]

File: repo/packages/K/kibana/1200/marathon.json.mustache (hunk 1 above)
Comment (kaiwalyajoshi) [reply repo/packages/K/kibana/1200/marathon.json.mustache]: Previous releases reference resource.json assets:
https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kibana/700/marathon.json.mustache#L59-L73

https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kibana/700/resource.json#L2-L7

Comment (rishabh96b) [reply repo/packages/K/kibana/1200/marathon.json.mustache]: The stub used was permanent only but somehow some random artifacts got rendered in the marathon as the version was old and its corresponding artifacts were already uploaded.

Comment (rishabh96b) [reply repo/packages/K/kibana/1200/marathon.json.mustache]: There is no need to worry as the resource.json has correct URLs. I have removed the rendered uris. Should be OK now.
https://github.com/mesosphere/universe/blob/automated/release_kibana_2.7.0-6.8.2_CTHrIg/repo/packages/K/kibana/1200/resource.json#L3-L6

**Review by kaiwalyajoshi — APPROVED** [review]

**Pull request merged**